stdlib/public/Concurrency/ExecutorAssertions.swift

```diff
@@ -32,9 +32,18 @@ extension SerialExecutor {
   /// * In `-O` builds (the default for Xcode's Release configuration), stops
   ///   program execution.
   ///
-  /// * In `-Ounchecked` builds, the optimizer may assume that this function is
-  ///   never called. Failure to satisfy that assumption is a serious
-  ///   programming error.
+  /// - Note: This check is performed against the actor's serial executor,
+  ///   meaning that / if another actor uses the same serial executor--by using
+  ///   that actor's serial executor as its own ``Actor/unownedExecutor``--this
+  ///   check will succeed , as from a concurrency safety perspective, the
+  ///   serial executor guarantees mutual exclusion of those two actors.
+  ///
+  /// - Parameters:
+  ///   - message: The message to print if the assertion fails.
+  ///   - file: The file name to print if the assertion fails. The default is
+  ///           where this method was called.
+  ///   - line: The line number to print if the assertion fails The default is
+  ///           where this method was called.
```

Comment:
```suggestion
  ///   - file: The file name to print if the assertion fails. The default value is
  ///           the file where this method was called.
  ///   - line: The line number to print if the assertion fails The default value is
  ///           the line where this method was called.
```

To avoid the misreading of of "where" telling you some location of the default value.